File: Pagination.java
import java.util.*;
public class Pagination
{
    public static void main(String[] args)
    {
        Scanner sc1= new Scanner(System.in);
        Scanner sc2= new Scanner(System.in);

        System.out.println("Total Number of Records ");
        int noOfRecords=sc1.nextInt();
        System.out.println("");
        System.out.print("Record per Page :");
        int recordsPerPage=sc1.nextInt();
        System.out.println("");
        int output=(int)(Math.ceil((double)noOfRecords/recordsPerPage));
        System.out.println("Output: Number of Pages : "+output);
        sc1.close();
        sc2.close();
    }
}

File: Pagination.js
var noOfRecords= 160;
var recordsPerPage= 40;
var output;
console.log("Program is Starting");

if(noOfRecords <=0 && recordsPerPage <=0)
{
    console.log("Number of records/ Records per page cannot be zero or negative")
}
else if(noOfRecords > recordsPerPage)
{
    console.log("Number of Records = "+noOfRecords);
    console.log("Records per Page = "+recordsPerPage);
    output=Math.ceil(noOfRecords/recordsPerPage);
    console.log("Number of Pages = "+output);
}
else{
    console.log("'Number of Pages = 1");
}
console.log("Program is Ending");